.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+<!---
+  This template provides some ideas of things to include in your PR description.
+  To start, try providing a short summary of your changes in the Title above.
+  If a section of the PR template does not apply to this PR, then delete that section.
+ -->
+
+## Motivation
+
+_(fill-in or delete this section)_
+
+<!--- 
+  What goal is this change working towards?
+  If this PR fixes one of more issues, list them here.
+  One line each, like so:
+    Fixes #123
+    Fixes #39
+-->
+
+## Changes
+
+_(fill-in or delete this section)_
+
+<!---
+  Provide a bullet pointed summary of how each file was changed.
+  Briefly explain any decisions you made with respect to the changes.
+-->
+
+## Testing
+
+_(fill-in or delete this section)_
+
+<!--- Describe how you tested this change. -->
+
+## Reviewer Guidelines
+
+_(fill-in or delete this section)_
+
+<!--
+   Is there any particular feedback you would / wouldn't like?
+   Which parts of the code should reviewers focus on?
+-->
+
+## Out of Scope / Future Work
```

Comment:
I think we should focus on getting dedicated features in within one PR, so I'm not sure if we need this section at all (open for discussion).

Reply:
I had the same thought in passing ^^ I'll remove this section 👍

Reply:
✅  https://github.com/urfave/cli/pull/963/commits/b2a974ead08b356b0644b4a9e8f7e13a1183c70b